.github/workflows/release-doc.yml

```diff
@@ -0,0 +1,19 @@
+name: Docs
+
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch: {}
+
+jobs:
+  release-docs:
+    permissions:
+      contents: write
```

Comment:
We're not doing github pages anymore, right? So we can drop `pages`. I don't believe we're pushing docs back to the repo either, so we can drop `contents`

Reply:
Based on gh-pages - probably we need `contents` but not `pages` ?